Documentation/English/Supported.md

```diff
@@ -0,0 +1,194 @@
+
+# Supported
+
+*Configurations of hardware & software that you can use.*
+
+<br>
+
+## Games
+
+<br>
+
+### **[Illusion]**
+
+<br>
+
+[<img
+    src = '../../Resources/Games/Koikatsu Sunshine.webp'
+    height = 60
+    title = 'Koikatsu Sunshine  |  コイカツ！サンシャイン'
+/>
+][Koikatsu Sunshine]                     
+![Badge VR]
+
+[<img
+    src = '../../Resources/Games/Koikatsu Party.png'
+    height = 60
+    title = 'Koikatsu Party  |  コイカツ Party'
+/>
+][Koikatsu Party]                          
+![Badge VR]
+
+[<img
+    src = '../../Resources/Games/Koikatsu.png'
+    height = 60
+    title = 'Koikatsu  |  コイカツ'
+/>
+][Koikatsu]                      
+![Badge VR]
+
+[<img
+    src = '../../Resources/Games/Honey Select 2.webp'
+    height = 60
+    title = 'Honey Select 2  |  ハニーセレクト２'
+/>
+][Honey Select 2]       
+![Badge VR]
+
+
+[<img
+    src = '../../Resources/Games/PlayHome.png'
+    height = 60
+    title = 'PlayHome  |  プレイホーム'
+/>
+][PlayHome]                          
+![Badge VR]
+
+
+<br>
+
+### **[Miconisomi]**
+
+<br>
+
+[<img
+    src = '../../Resources/Games/Houkago Rinkan Chuudoku.png'
+    height = 150
+    title = 'Houkago Rinkan Chuudoku'
+/>
+][Houkago Rinkan Chuudoku]
+
+[![Badge AGHVR]][AGHVR]
+
+[<img
+    src = '../../Resources/Games/Insult Order.png'
+    height = 60
+    title = 'Insult Order'
+/>
+][Insult Order]
+
+[![Badge IOVR]][IOVR]
+
+<br>
+
+### **[Kiss]**
+
+<br>
+
+[<img
+    src = '../../Resources/Games/Custom Order Maid 3D 2.png'
+    height = 60
+    title = 'Custom Order Maid 3D 2'
+/>
+][Custom Order Maid 3D 2]       
+![Badge VR]
+
+<br>
+
+### **[Momoiro]**
+
+<br>
+
+[<img
+    src = '../../Resources/Games/Our Apartment.jpg'
+    height = 60
+    title = 'Custom Order Maid 3D 2'
+/>
+][Our Apartment]
+
+<br>
+<br>
+
+## Devices
+
+**LoveMachine** relies on the **[Buttplug.io]** project <br>
```

Comment:
I make massive edits to these docs all the time, so any attempt at manual word wrapping is a waste of time.
The intended way of reading the docs is Github Pages. If you'd like to help with readability, the best way would be to set an appropriate Jekyll theme. I experimented with some, but then I gave up.

Reply:
In case you could separate the user side (the github pages pages) from the dev side (the repo page) by using either a `/docs/` folder, a separate branch or separate repo, the first approach would of course be easiest.

Not sure if you know, but `/docs/` is the only allowed subfolder name for github pages sites.

Reply:
The repo page also displays the "user side" readme, and that is intentional. I see no need to separate the two.

Reply:
Why would you use a github pages page in the first place then?

Reply:
It's distraction-free and I can change the theme as I see fit.